crates/test-util/src/wast.rs

```diff
@@ -430,125 +486,188 @@ impl WastTest {
-            ];
+            #[cfg(target_arch = "aarch64")]
+            {
+                let unsupported = [
+                    // Externref / GC related.
+                    "extended-const/elem.wast",
+                    "extended-const/global.wast",
+                    "spec_testsuite/br_table.wast",
+                    "spec_testsuite/global.wast",
+                    "spec_testsuite/linking.wast",
+                    "spec_testsuite/ref_func.wast",
+                    "spec_testsuite/ref_is_null.wast",
+                    "spec_testsuite/ref_null.wast",
+                    "spec_testsuite/select.wast",
+                    "spec_testsuite/table_fill.wast",
+                    "spec_testsuite/table_get.wast",
+                    "spec_testsuite/table_grow.wast",
+                    "spec_testsuite/table_set.wast",
+                    "spec_testsuite/table_size.wast",
+                    "misc_testsuite/table_grow_with_funcref.wast",
+                    "misc_testsuite/component-model/modules.wast",
+                    "misc_testsuite/winch/table_get.wast",
+                    "misc_testsuite/winch/table_set.wast",
+                    "misc_testsuite/winch/table_fill.wast",
+                    "misc_testsuite/linking-errors.wast",
+                    "misc_testsuite/no-panic.wast",
+                    "misc_testsuite/externref-id-function.wast",
+                    "misc_testsuite/simple_ref_is_null.wast",
+                    "misc_testsuite/mutable_externref_globals.wast",
+                    "misc_testsuite/many_table_gets_lead_to_gc.wast",
+                    "misc_testsuite/no-mixup-stack-maps.wast",
+                    "misc_testsuite/externref-segment.wast",
+                    "misc_testsuite/externref-segments.wast",
+                    "misc_testsuite/externref-table-dropped-segment-issue-8281.wast",
+                    // Known bugs that don't cause segfaults.
+                    "spec_testsuite/call_indirect.wast",
+                    "spec_testsuite/elem.wast",
+                    "spec_testsuite/f32_cmp.wast",
+                    "spec_testsuite/f64_cmp.wast",
+                    "spec_testsuite/func_ptrs.wast",
+                    "spec_testsuite/i64.wast",
+                    "spec_testsuite/if.wast",
+                    "spec_testsuite/imports.wast",
+                    "spec_testsuite/local_set.wast",
+                    "spec_testsuite/local_tee.wast",
+                    "spec_testsuite/loop.wast",
+                    "spec_testsuite/table_copy.wast",
+                    "spec_testsuite/table_init.wast",
+                    "misc_testsuite/custom-page-sizes/custom-page-sizes.wast",
+                    "misc_testsuite/winch/table_grow.wast",
+                    "spec_testsuite/proposals/custom-page-sizes/custom-page-sizes.wast",
+                    "misc_testsuite/memory64/more-than-4gb.wast",
+                    "misc_testsuite/call_indirect.wast",
+                ];
```

Comment:
Could the platform-independent tests, the ones that fail on all platforms, be deduplicated between here and x64?

Reply:
Sure, yeah. Added that in https://github.com/bytecodealliance/wasmtime/pull/10750/commits/0d4784bc61f6b7a5350b2e91d0ec7df85880cae3